test/cql-pytest/nodetool.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2021 ScyllaDB
+#
+# This file is part of Scylla.
+#
+# Scylla is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Scylla is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Scylla.  If not, see <http://www.gnu.org/licenses/>.
+##################################################################
+
+# This file provides a few nodetool-like commands that may be useful
+# for tests. The intent is *not* to test nodetool itself, but just to
+# use its functionality to test CQL. So when testing against a locally-
+# running Scylla these functions do not actually use nodetool but rather
+# the REST API. This simplifies running the test because an external Java
+# process implementing JMX is not needed. But when the REST API port is
+# not available (running against Cassandra or some remote installation of
+# Scylla) "nodetool" *is* used, and must be available in the path or
+# chosen with the NODETOOL environment variable.
+
+import requests
+import os
+import subprocess
+
+# For a "cql" object connected to a particular node, find the REST API URL
+# with the same node and port 10000.
+def rest_api_url(cql):
+    return f'http://{cql.cluster.contact_points[0]}:10000'
```

Comment:
The commitlog suggests that `cql` alows for sending the command to a specific node, but here I see that it actually represents the entire cluster and we just always select the first node from it. So I don't understand how we specify which node to talk to with `cql` parameter.
This doesn't have to be addressed now as we only support a single node at the moment, just wanted to note it.

Reply:
I was thinking that in the future **maybe** we'll want "cql" to somehow connected to one specific node in a bigger cluster, similar to how we have in dtest sessions which are configured to always send request to the same node. You're right that maybe this function will need to change. I think it will become clearer what we need to do when we really do anything with multiple nodes.

Reply:
I agree, we don't need to sort this out this now.